src/components/Person/Person.jsx

```diff
@@ -1 +1,33 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+import '../../App.scss';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    isMarried,
+    partnerName,
+    sex,
+  } = person;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
+
+      {age && <p className="Person__age">{`I am ${age}`}</p>}
+
+      {isMarried
+        ? (
+          <p className="Person__partner">
+            {
+              sex === 'm'
+                ? `${partnerName} is my wife`
+                : `${partnerName} is my husband`
+            }
```

Comment:
just move this to the variable
```suggestion
            {`${partnerName} is my ${partner}`}
```

Reply:
Done